pkgs/development/tools/analysis/radare2/default.nix

```diff
@@ -37,6 +38,14 @@ stdenv.mkDerivation rec {
     sha256 = "0n3k190qjhdlj10fjqijx6ismz0g7fk28i83j0480cxdqgmmlbxc";
   };
 
+  patches = [
+    # fix for CVE-2021-32613
+    (fetchpatch {
```

Comment:
Could you actually give the patch a `name = ` of something like `CVE-2021-32613.patch`? This can be detected by some automated vuln checker tools.